modules/distribution/pom.xml

```diff
@@ -31,26 +31,32 @@
     <dependency>
       <groupId>org.apache.fluo</groupId>
       <artifactId>fluo-accumulo</artifactId>
+      <optional>true</optional>
```

Comment:
Because if anybody had a dependency on the tar.gz artifact from Maven, they won't have transitive dependencies on the tarball's dependencies.